src/ui/public/kbn_top_nav/kbn_top_nav.js

```diff
@@ -35,18 +35,21 @@
 
 import _ from 'lodash';
 import angular from 'angular';
+import { customMenu } from 'ui/kibiter/menu/custom_menu'
```

Comment:
Organize these imports along with the style one in a single block to make clear they only belong to kibiter menu functionality.